modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,199 @@
+
+== Prerequisites
+
+include::shared:partial$enterprise-license.adoc[]
+
+. To check if you already have a license key applied to your cluster:
++
+[,bash]
+----
+rpk cluster license info
+----
+
+. Enable xref:manage:tiered-storage.adoc#set-up-tiered-storage[Tiered Storage] for the topics for which you want to generate Iceberg tables.
+. Install `rpk`.
+
+== Limitations
+
+* You can only enable the topic integration for new Redpanda topics. 
+* It is not possible to append or backfill data from Redpanda topics to an existing Iceberg table.
+* JSON schemas are not currently supported. For Avro schemas, records cannot contain fields greater than 128KB.
+* You can only use one schema per topic. Schema versioning as well as upcasting (where a value is cast into its more generic data type) are not supported. 
+
+== Architecture 
+
+Apache Iceberg is an open source format specification for defining structured tables in a glossterm:data lake[]. The table format enables you to easily and quickly manage, query, and process huge amounts of structured and unstructured data, like you would manage and run SQL queries against relational data in a database or data warehouse. The open format allows many different languages, tools, and applications to process the same data in a consistent way, so you can avoid vendor lock-in. This data management system is also referred to as a _data lakehouse_.
+
+In the Iceberg specification, tables comprise the following:
+
+* Data layer
+** Data files: Store the actual data. The Redpanda integration currently supports the Parquet file format. Parquet files are column-based and suitable for analytical workloads at scale, and come with compression capabilities that optimize files for object storage. 
+* Metadata layer
++
+--
+** Manifest files: Track data files and contain metadata about those files, such as record count, partition membership, and file paths.
+** Manifest list: Tracks all the manifest files belonging to a table, including file paths and upper and lower bounds for partition fields. 
+** Metadata file: Stores metadata about the table, including its schema, partition information, and snapshots. Whenever a change is made to the table, a new metadata file is created and becomes the latest version of the metadata in the catalog.
+--
++
+In the Redpanda Iceberg integration, the manifest files are in JSON format.
+* Catalog: Contains the current metadata pointer for the table. Clients reading and writing data to the table see the same version of the current state of the table. You'll configure your Iceberg catalog to point to your object storage bucket or container where the Redpanda data in Iceberg format is located. Redpanda uses the https://iceberg.apache.org/concepts/catalog/#catalog-implementations[Iceberg REST catalog^] endpoint to update your catalog when there are changes to the Iceberg data and metadata.
+
+image::shared:iceberg-integration.png[]
```

Comment:
I know there's probably not time for this but I could do with some numbering on this diagram.

Reply:
@mattschumpert does numbering in the provided diagram (same as the one used in the blog post from a while back)? Is there anything else we should add to the design request in [Monday](https://redpanda.monday.com/boards/2297694569/pulses/7628352596)?

Reply:
makes sense to add numbers